docs/start.md

```diff
@@ -6,204 +6,196 @@
 ---
 ## SDK 下载
 
-[![download_sdk](../static/download-sdk.png)](../demo/android-sbrc-sdk-release-2015-03-05-2e0633d-with-demo.zip)
+(如使用 Android Studio 毋须下载)
+
+[![download_sdk](../static/download-sdk.png)](../maven/hihex/sbrc/sbrc/1.6.2/sbrc-1.6.2.aar)
 
 ## SDK 环境要求
 
-| 环境要求     |                                                              |
-| -----------: | ------------------------------------------------------------ |
-| Android 系统 | Android 4.1（API 15）或更高，使用 x86 或 ARMv7（不支持MIPS） |
-| JDK 版本     | 不低于 1.6                                                   |
-| IDE          | Eclipse with ADT 22.3 或更高，Android Studio 1.1 或更高      |
+| 环境要求     |                                                                   |
+|-------------:|-------------------------------------------------------------------|
+| Android 系统 | Android 4.1（API 15）或更高，使用 x86 或 ARMv7（不支持MIPS）      |
+| JDK 版本     | 不低于 1.6                                                        |
+| IDE          | Android Studio 1.3 beta 或更高、Eclipse Mars 及 ADT 23.0.6 或更高 |
 
 
 ## 接入 SDK
 
-1. 解压下载好的 android-sbrc-sdk-release-*.zip。
-
-2. 将 SDK 的 `libs/` 文件夹与项目的`libs/`文件夹合并。
-
-3. 放置 `android-sbrc.jar`：
+### 如果您使用 Gradle (Android Studio)
 
-   - ant 项目置于 `libs/` 下
+1. 在属于该模组的 `build.gradle` 加上以下代码：
 
-   - gradle 项目置于 `app/libs/` 下,并在 `build.gradle` 下确保:
-     
-     ```json
-        android {
-            ...
-            sourceSets {
-                main {
-                    jniLibs.srcDirs = ['libs']
-                }
+        repositories {
+            maven {
+                url 'http://dev.hihex.com/maven'
             }
-            ...
         }
-
         dependencies {
-            compile files('libs/android-sbrc.jar')
+            compile 'hihex.sbrc:sbrc:1.6.2@aar'
         }
-     ```
-
-4. 将 SDK 的 `res/` 文件夹与项目的 `res/` 文件夹合并。
 
-5. IDE 刷新后即可生效。
+2. 按一下 “Sync Project with Gradle Files” 让专案与 Gradle 设定同步。
 
-   ![libs_image](../static/libs.png)
+### 如果您使用 Eclipse ADT…
 
-## 更新 AndroidManifest.xml
+> 亲，Google 已经宣布[不再支援 Eclipse ADT 了](http://android-developers.blogspot.hk/2015/06/an-update-on-eclipse-android-developer.html)，我们也不会支持多久了，赶紧换 IDE 吧
 
-1. 在 `manifest` 标签下加入以下权限:
+1. 先安装 Python 3。
+2. [下载 SDK](../maven/hihex/sbrc/sbrc/1.6.2/sbrc-1.6.2.aar)
+3. 下载 `aar-to-eclipse`：
```

Comment:
@rexxarM Yeah we need to put the "manual installation" part in the SDK_README.md somewhere, though I don't prefer to put it right into the start page. It makes the page too long.

Alternatively we could add a Tcl/Tk GUI and `py2exe` the script for Windows.

(We also need to put the Javadoc (all are static HTML/CSS/JS/PNG) somewhere.)